src/glossary.rst

```diff
@@ -5535,6 +5671,14 @@ union value
 :dp:`fls_9BPrxky3a4nE`
 A :dt:`union value` is a :t:`value` of a :t:`union type`.
 
+.. _fls_Is9hWLC6Q0g5:
+
+unique immutable borrow
+^^^^^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_eXrivAmNxzmv`
+HELP LUKAS!
```

Comment:
Though one :) https://doc.rust-lang.org/stable/reference/types/closure.html#unique-immutable-borrows-in-captures

Reply:
Added a small definition of this in 0a5bbdd